### PR TITLE
NewNavigation: Fix default pinned items

### PR DESCRIPTION
**Pull request by kaydelaney**
**What this PR does / why we need it**:
Adds missing items to the list of nav items pinned by default.

### CODE REVIEW

File: public/app/core/reducers/navBarTree.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { NavModelItem } from '@grafana/data';
 import config from 'app/core/config';
 
-const defaultPins = ['home', 'dashboards', 'explore', 'alerting'].join(',');
+const defaultPins = ['home', 'dashboards', 'explore', 'alerting', 'cfg', 'admin', 'profile', 'help'].join(',');
```
Comment (Clarity-89) [inline public/app/core/reducers/navBarTree.ts:5]: Does this mean that Enterprise features, e.g. reporting won't be pinned by default?

**Review by ashharrison90 — COMMENTED** [review]
can we make the default pins just default to everything?

File: public/app/core/reducers/navBarTree.ts (hunk 1 above)
Comment (kaydelaney) [reply public/app/core/reducers/navBarTree.ts:5]: Good point! Let me think of something

**Review by ashharrison90 — APPROVED** [review]
lgtm 👍  this will change very soon anyway

**Review by Clarity-89 — APPROVED** [review]

**Pull request merged**